apps/atomy-q/API/app/Adapters/Ai/ProviderComparisonAwardClient.php

```
@@ -0,0 +1,94 @@
+     */
+    public function comparisonOverlay(ComparisonOverlayRequest $request): ComparisonOverlayResponse
+    {
+        $payload = $request->toPayload();
+
+        try {
+            return new ComparisonOverlayResponse($this->transport->invoke(AiStatusSchema::ENDPOINT_GROUP_COMPARISON_AWARD, [
+                ...$payload,
+                'action' => 'comparison_overlay',
+            ]));
+        } catch (\Throwable $e) {
+            throw ComparisonAwardAiException::fromThrowable($e, 'comparison_overlay', $payload);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Adapters\Ai\Exceptions\ComparisonAwardAiException' in method 'comparisonOverlay'. (undefined)

(StaticAccess)